beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -883,12 +882,22 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
                             failed_columns.insert(*column);
                             failed_peers.insert(*peer);
                         }
-                        for peer in failed_peers.iter() {
-                            network.report_peer(*peer, *action, "failed to return columns");
-                        }
                         // Retry the failed columns if the column requests haven't exceeded the
                         // max retries. Otherwise, remove treat it as a failed batch below.
                         if !*exceeded_retries {
+                            // Set the batch back to `AwaitingDownload` before retrying.
+                            // This is to ensure that the batch doesn't get stuck in `Downloading` state.
+                            //
+                            // DataColumn retries has a retry limit so calling `downloading_to_awaiting_download`
+                            // is safe.
+                            if let BatchOperationOutcome::Failed { blacklist } =
```

Comment:
`BatchOperationOutcome::Failed` is no longer reachable after changing the call  to `downloading_to_awaiting_download`.

Also i think the function name conveys clearly what it does, but not its intended usage, but I also struggle to come up with something better. 

`batch.download_failed_skip_attempt_count` seems a bit too wordy